lambdas/data-migration2/src/storeErrors.ts

```diff
@@ -0,0 +1,34 @@
+import moment from 'moment';
+
+const { s3 } = require('@cumulus/aws-client/services');
+const fs = require('fs');
+
+/**
+ * Store migration errors JSON file on S3.
+ *
+ * @param {Object} params
+ * @param {string} params.bucket - Name of S3 bucket where file will be uploaded
+ * @param {string[]} params.filepath - Write Stream file path
+ * @param {string} params.migrationName - Name of migration
+ * @param {string} params.stackName - User stack name/prefix
+ * @param {string | undefined} params.timestamp - Timestamp for unit testing
+ * @returns {void}
+ */
+export const storeErrors = async (params: {
+  bucket: string,
+  filepath: string,
+  migrationName: string,
+  stackName: string,
+  timestamp?: string,
+}) => {
+  const { bucket, filepath, migrationName, stackName, timestamp } = params;
+  const fileKey = `data-migration2-${migrationName}-errors`;
+  const dateString = timestamp || moment.utc().format('YYYY-MM-DD_HH:MM:SS');
+  const key = `${stackName}/${fileKey}_${dateString}.json`;
```

Comment:
Nit: let's be consistent in our use of dashes vs underscores

```suggestion
  const key = `${stackName}/${fileKey}-${dateString}.json`;
```

Reply:
https://github.com/nasa/cumulus/pull/2179/files#diff-56c5b1137921652643573bffebcd7fae07a919d6517811b49ab0f84759ba10beR56